themes/theme-gmd/extension-config.json

```diff
@@ -238,14 +238,22 @@
         "colors": {
           "background": "#f8f8f8",
           "light": "#fff",
+          "lightTransparent": "rgba(255, 255, 255, 0.82)",
           "dark": "#000",
+          "lightDark": "#323232",
           "placeholder": "#f2f2f2",
+          "overlay": "#fafafa",
+          "lightOverlay": "#fafafa",
+          "darkOverlay": "#f0f0f0",
           "primary": "#fa5400",
           "primaryContrast": "#fff",
           "accent": "#5ccee3",
           "accentContrast": "#fff",
           "cta": "#fa5400",
           "ctaContrast": "#fff",
+          "gray": "#747474",
+          "lightGray": "#9a9a9a",
```

Comment:
This is the same as `shade3`. So, either change all usages of `shade3` to this one or the other way around, please.